tests/MongoDB.Driver.Tests/Specifications/crud/ReplaceOneTest.cs

```diff
@@ -65,11 +65,15 @@ protected override ReplaceOneResult ExecuteAndGetResult(IMongoDatabase database,
         {
             if (async)
             {
+#pragma warning disable 618
```

Comment:
Would it be better to update this test to use ReplaceOneOptions instead of UpdateOptions?

Reply:
done